modules/Activities/tpls/PopupHeader.tpl

```diff
@@ -0,0 +1,83 @@
+* IN WHICH THE COPYRIGHT IS OWNED BY SUGARCRM, SUGARCRM DISCLAIMS THE WARRANTY
+* OF NON INFRINGEMENT OF THIRD PARTY RIGHTS.
+*
+* This program is distributed in the hope that it will be useful, but WITHOUT
+* ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+* FOR A PARTICULAR PURPOSE.  See the GNU Affero General Public License for more
+* details.
+*
+* You should have received a copy of the GNU Affero General Public License along with
+* this program; if not, see http://www.gnu.org/licenses or write to the Free
+* Software Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+* 02110-1301 USA.
+*
+* You can contact SugarCRM, Inc. headquarters at 10050 North Wolfe Road,
+* SW2-130, Cupertino, CA 95014, USA. or at email address contact@sugarcrm.com.
+*
+* The interactive user interfaces in modified source and object code versions
+* of this program must display Appropriate Legal Notices, as required under
+* Section 5 of the GNU Affero General Public License version 3.
+*
+* In accordance with Section 7(b) of the GNU Affero General Public License version 3,
+* these Appropriate Legal Notices must retain the display of the "Powered by
+* SugarCRM" logo and "Supercharged by SuiteCRM" logo. If the display of the logos is not
+* reasonably feasible for  technical reasons, the Appropriate Legal Notices must
+* display the words  "Powered by SugarCRM" and "Supercharged by SuiteCRM".
+*/
+*}
+<!DOCTYPE HTML>
+<html {$langHeader}>
+<head>
+
+    {$ieCompatMode}
+    <meta http-equiv="Content-Type" content="text/html; charset="{$charset}">
+
+    <link href="themes/{$theme}/css/normalize.css" rel="stylesheet">
+    <link href="themes/{$theme}/css/bootstrap.min.css" rel="stylesheet">
+    <link href="themes/{$theme}/css/fonts.css" rel="stylesheet">
+    <link href="themes/{$theme}/css/grid.css" rel="stylesheet">
+    <link href="themes/{$theme}/css/footable.core.css" rel="stylesheet">
+    <link href="themes/{$theme}/css/style.css" rel="stylesheet">
+
+    <script type="text/javascript" src="cache/include/javascript/sugar_grp1_jquery.js"></script>
+    <script type="text/javascript" src="cache/include/javascript/sugar_grp1_yui.js"></script>
+    <script type="text/javascript" src="cache/include/javascript/sugar_grp1.js"></script>
+
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="initial-scale=1.0, user-scalable=no"/>
+
+
+</head>
+<body class="popup">
+<table width="100%" cellpadding="0" cellspacing="0">
+    <tbody>
+    <tr>
+        <td>
+            <div style="text-align: right">
+                <span class="glyphicon glyphicon-print glyph-print"></span>
+                <a href='javascript:print();' class='utilsLink'>Print </a>
+            </div>
+
+            <div class="moduleTitle">
+                <div>
+                    {$title}
+                </div>
+            </div>
+
+        </td>
+    </tr>
+    </tbody>
+</table>
+</body>
```

Comment:
Also in your screen shot, please add some spacing to the left of title, and to the right of the print link.